lib/tasks/testing.rake

```diff
@@ -1,6 +1,6 @@
 namespace :test do
   desc 'Shortcut to run all ruby and javascript linters and specs'
-  task :all do
+  task all: :environment do
```

Comment:
I'm curious what motivated this. Since some of the other commands probably _also_ load the environment, I'd guess it's happening at least twice now.

Reply:
This was in response to a new Rubocop warning, but yeah — it's probably not necessary since we're just shelling out to a bunch of commands in this task rather than actually doing anything with the loaded environment. 

Will remove!